cypress/integration/plugins/dashboards-assistant/chatbot_interaction_trace_spec.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BASE_PATH } from '../../../utils/constants';
+
+if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
+  describe('Interaction trace spec', () => {
+    before(() => {
+      // Set welcome screen tracking to false
+      localStorage.setItem('home:welcome:show', 'false');
+      // Set new theme modal to false
+      localStorage.setItem('home:newThemeModal:show', 'false');
```

Comment:
Just like @kavilla said before, shall we restore the localStorage after all tests finished.

Reply:
that should be nice to have. Normally, each test case should have its own setup in before hook to setup the expected state before start actual test.

Reply:
clean up added